codice/alliance
# CAL 457 pom-fix-run fails when run after the project version is increased

#### What does this PR do?
Addresses checkstyle related issues that were causing pom-fix-run to fail when run on its own and breaking the Alliance CI pipeline.

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
@bdeining 
@djblue  

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@shaundmorris

#### How should this be tested?
Successful CI pipeline build.
Also, should make sure that pom-fix still identified discrepancies between pom and feature files.

#### Any background context you want to provide?

#### What are the relevant tickets?

[CAL-457](https://codice.atlassian.net/browse/CAL-457)
#### Screenshots (if appropriate)
#### Checklist:
